docs/content/preview/yugabyte-voyager/reference/yb-voyager-cli.md

```diff
@@ -64,7 +64,7 @@ The valid *arguments* for export schema are described in the following table:
 
 | Argument | Description/valid options |
 | :------- | :------------------------ |
-| [-e, --export-dir](#export-dir) <path> | Path to the directory where the data files will be exported. |
+| [-e, --export-dir](#export-dir) <path> | Path to the directory which is a workspace used to keep the exported  schema, data, state, and logs.|
```

Comment:
Everywhere
Path to the export directory. This directory is a workspace used to keep the exported schema, data, state, and logs.